packages/cli-repl/src/smoke-tests.ts

```diff
@@ -68,13 +67,13 @@ export async function runSmokeTests(smokeTestServer: string | undefined, executa
     output: /Test succeeded/,
     includeStderr: false,
     exitCode: 0,
-    testArgs: [smokeTestServer as string]
```

Comment:
`--fix` removed these (along with a lot of as any that I since reverted) because it thinks these aren't necessary. You can tell by the space it left. Which turned out to be handy for finding these with find/replace :) Prettier will clean that up soon enough. I could undo these just like I did all the `as any` removals, but none of them have caused issues yet so I guess it really was unnecessary.